app/src/main/java/org/greenstand/android/TreeTracker/view/BorderedTextField.kt

```diff
@@ -51,4 +62,11 @@ fun BorderedTextField(
             keyboardActions = keyboardActions
         )
     }
+
+    DisposableEffect(Unit) {
```

Comment:
Since we don't need to do any cleanup, we can use 'LaunchedEffect' instead.

Reply:
i see the difference, thanks!